Example/Controllers/Extensions/ViewController+MiniApp.swift

```diff
@@ -34,9 +34,18 @@ extension ViewController: MiniAppNavigationDelegate {
                     self.currentMiniAppInfo = responseData
                     self.fetchMiniApp(for: responseData)
                 case .failure(let error):
+                    var message: String
+                    switch error {
+                    case .noPublishedVersion:
+                        message = "No published versions for the provided Mini App ID."
+                    case .miniAppNotFound:
+                        message = "No Mini App found for the provided Mini App ID."
```

Comment:
if we localized the regular message maybe we can also localize these two

Reply:
I guess this is for the sample app, so I guess it's fine.

Reply:
I localized these.